homeassistant/components/emulated_roku/strings.json

```diff
@@ -5,10 +5,10 @@
     "step": {
       "user": {
         "data": {
-          "advertise_ip": "Advertise IP",
-          "advertise_port": "Advertise port",
-          "host_ip": "Host IP",
-          "listen_port": "Listen port",
+          "advertise_ip": "Advertise [%key:common::config_flow::data::ip%]",
```

Comment:
We try to avoid mixing in references in a sentence.

Reply:
👍 Reverted

Reply:
👍 Removed references, although edited the values directly to bring consistency (Ex: IP-->IP Address)